docs/user-guide/cli-configure-scs-on-headless-linux-os.md

```diff
@@ -2,12 +2,22 @@
 
 Perform the following configurations on headless and z/Linux operating systems.
 
+:::note
+
+For CI/CD pipelines we recommend using the credential management provided by the CI/CD tool (for example, Jenkins credentials or GitHub secrets) to store credentials and load them into environment variables in the pipeline. See [Using environment variables](../user-guide/cli-using-using-environment-variables.md#store-credentials-securely-in-cicd-pipelines) for more information.
```

Comment:
Add comma
For CI/CD pipelines, we recommend

Reply:
addressed in [PR 3580](https://github.com/zowe/docs-site/pull/3580)